x/erc20/keeper/params.go

```diff
@@ -24,6 +24,15 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 
 // SetParams sets the erc20 parameters to the param space.
 func (k Keeper) SetParams(ctx sdk.Context, params types.Params) error {
+	// if a precompile is disabled or deleted in the params, we should remove the codehash
+	oldDynamicPrecompiles := k.getDynamicPrecompiles(ctx)
+	disabledPrecompiles := types.GetDisabledPrecompiles(oldDynamicPrecompiles, params.DynamicPrecompiles)
+	for _, precompile := range disabledPrecompiles {
+		if err := k.UnRegisterERC20CodeHash(ctx, precompile); err != nil {
+			return err
+		}
+	}
```

Comment:
are we covered also for the case where we enable back a precompile? Or should we add that logic
Will be the case of re-enabling an already existing erc20 precompile